website/frontend/src/pages/OurSolutions/ResearchPage.js

```diff
@@ -118,19 +118,18 @@ const ResearchPage = () => {
         <SEO
           title="Our Solutions"
           siteTitle="For Research"
-          description="With the exciting air quality data, we actively collaborate with partners across Africa to jointly tackle air quality research challenges."
+          description="AirQo actively collaborates with researchers across the world to jointly tackle air quality research challenges."
```

Comment:
I can seen some content changes, perhaps we need to also include this in the PR description. Thanks.